### PR TITLE
Changes for starboard upgrade to 0.15.4 version

### PR DESCRIPTION
**Pull request by anupamtamrakar**
Deployment changes for starboard upgrade.

@KoppulaRajender  can you please make sure that these changes should also go in aqua helm and other KE deployment tools

cc @deven0t

### CODE REVIEW

**Comment by deven0t** [comment]
@KoppulaRajender  Please wait for a new update release(for 2022.4) from release team. Only after that we should merge this.

**Review by deven0t — APPROVED** [review]
LGTM (y)

**Review by KoppulaRajender — APPROVED** [review]

**Pull request merged**
